test/Microsoft.Framework.Runtime.Tests/ProjectFilesCollectionFacts.cs

```diff
@@ -59,6 +59,30 @@ public void FilePatternsWorkForArraysAreSet()
             Assert.Equal(new[] { "a.cs", @"foo.js" }, target.ResourcesPatterns);
         }
 
+
+        [Fact]
+        public void FilePatternsDoNotIncludeComments()
+        {
+            var rawProject = Deserialize(@"
+{
+    ""code"": [""*.cs"", /* FOO */ ""../*.cs""],
+    ""exclude"": [""buggy/*.*"", /* FOO */],
+    ""bundleExclude"": [""no_pack/*.*"", /* FOO */],
+    ""preprocess"": [""other/**/*.cs"", /* FOO */ ""*.cs"", ""*.*""],
+    ""shared"": [""shared/**/*.cs;../../shared/*.cs"", /* FOO */],
+    ""resources"": [""a.cs"", /* FOO */ ""foo.js""]
+}");
+
+            var target = new ProjectFilesCollection(rawProject, string.Empty);
+
+            Assert.Equal(new[] { "*.cs", @"../*.cs" }, target.SourcePatterns);
```

Comment:
Kind of funny that these use `@"foo"` string literals even though there's nothing to escape here (I see it's just copied from other tests).